Dockerfile

```diff
@@ -49,18 +48,19 @@ RUN \
 # copy sources
 COPY . /home/jhipster/generator-jhipster
 
+# When https://github.com/docker/docker/pull/28499 will be merged and released,
+# remove the following line and add "--chown jhipster:jhipster" to the previous one
 RUN \
   # fix jhipster user permissions
-  chown -R jhipster:jhipster \
-    /home/jhipster \
-    /usr/lib/node_modules && \
```

Comment:
Without that, the user can't use `npm install -g yarn` and update Yarn.

I know, it's very minor and the user will probably never do it, but I prefer keeping the change owner of `/usr/lib/node_modules/` to prevent some strange use cases.

```
jhipster@d0ab44b6078e:~$ npm install -g yarn
npm WARN checkPermissions Missing write access to /usr/lib/node_modules/yarn
npm WARN checkPermissions Missing write access to /usr/lib/node_modules
/usr/lib
`-- yarn@0.21.3 

npm ERR! Linux 4.8.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "yarn"
npm ERR! node v6.10.0
npm ERR! npm  v4.3.0
npm ERR! path /usr/lib/node_modules/yarn
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access

npm ERR! Error: EACCES: permission denied, access '/usr/lib/node_modules/yarn'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, access '/usr/lib/node_modules/yarn'
npm ERR!     at Error (native)
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/lib/node_modules/yarn' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/jhipster/.npm/_logs/2017-03-05T11_30_48_462Z-debug.log
```